A2rchi/chains/chain.py

```diff
@@ -29,16 +28,18 @@ def __init__(self):
         self.embedding_model = embedding_class_map[embedding_name]["class"](**embedding_class_map[embedding_name]["kwargs"])
 
         if self.utils_config["data_manager"]["use_HTTP_chromadb_client"]:
-            self.client = chromadb.HttpClient(host = self.utils_config["data_manager"]["chromadb_host"],
-                                              port = self.utils_config["data_manager"]["chromadb_port"])
+            self.client = chromadb.HttpClient(
+                host=self.utils_config["data_manager"]["chromadb_host"],
+                port=self.utils_config["data_manager"]["chromadb_port"],
+            )
         else:
             self.client = chromadb.PersistentClient(path = self.global_config["LOCAL_VSTORE_PATH"])
 
         self.collection_name = self.utils_config["data_manager"]["collection_name"] + "_with_" + embedding_name
         self.vectorstore = Chroma(client = self.client, collection_name = self.collection_name,
                                   embedding_function = self.embedding_model)
 
-        print(" N Colls: ",self.client.get_collection(self.collection_name))
+        print(" N Collection: ",self.client.get_collection(self.collection_name))
```

Comment:
Not sure what "N Collection" refers to here. This print statement outputs the name and some meta info about the collection we are using, as follows:

`name='dev_collection_with_OpenAIEmbeddings' id=UUID('bace30e9-e225-4aed-b70c-63844981ce91') metadata=None`


So maybe we should change the string at the beginning of the print statement to be something like "INFO - collection used: "

Reply:
Ahh okay, I was a bit confused about the print statement as well and tried making it more verbose, but your suggestion looks much better to me.

Reply:
I'll also take a second pass through the code in a subsequent PR to swap print statements for `logging.info` statements, which will have that `INFO` prefix by default